app/views/episode_metrics/_overview.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="container">
```

Comment:
specific to the episode metrics view. this is the overview section in the design that houses some episode-specific "quick info" of sorts.